src/client/components/app/Transactions/Labs/LabDepositTx.tsx

```diff
@@ -68,19 +67,17 @@ export const LabDepositTx: FC<LabDepositTxProps> = ({ onClose }) => {
   const sellTokensOptions = selectedLab
     ? [selectedLab.token, ...userTokens.filter(({ address }) => address !== selectedLab.token.address)]
     : userTokens;
-  const sellTokensOptionsMap = keyBy(sellTokensOptions, 'address');
-  let selectedSellToken: TokenView | undefined = sellTokensOptionsMap[selectedSellTokenAddress ?? ''];
+  const selectedSellToken = getSelectedSellToken({
+    sellTokensOptions,
+    selectedSellTokenAddress,
+    nativeCurrency: ETHEREUM_ADDRESS,
```

Comment:
Are we planning to use a different `nativeCurrency`? If not, I'd just use the `ETHEREUM_ADDRESS` directly in the `getSelectedSellToken()`